setup/gcp/installer/util/utils.py

```diff
@@ -302,6 +302,248 @@ def sanitize_conf_values(conf_values):
     return conf_values
 
 
+def merge_object(base_obj, target_obj,
```

Comment:
I really feel that we should get the naming correct here, as it will go a long way to improve understandability of the code.  Basically, we have `merge_from` and `merge_to`.  Can we call the args by that?

Or, how about `merge_source` and `merge_target`, with `target` being the opposite of you are actually using?  But I think would make more sense in this case, since `target` is what we are merging go.

nit: it would also be helpful to re-order the arg positions, so that their position convey the action sequence from left to right.
```
def merge_object(merge_from, merge to)
```

Reply:
Moved the functionality out to merge_engine.py, updated the naming of the params as suggested.